superset/datasets/commands/importers/v0.py

```diff
@@ -97,9 +76,6 @@ def import_dataset(
     if isinstance(i_datasource, SqlaTable):
         lookup_database = lookup_sqla_database
         lookup_datasource = lookup_sqla_table
-    else:
-        lookup_database = lookup_druid_cluster
-        lookup_datasource = lookup_druid_datasource
```

Comment:
Let's leave the `else` and raise an exception inside the block, otherwise we would end with undefined variables in lines 107, 108.